### PR TITLE
Fix docsify vulnerability

### PR DESCRIPTION
**Pull request by alexandre-lelain**
Hello,

First of all, thank you for making this awesome library. I'm using it on a project and it really rocks.

# The problem

There was a vulnerability spotted in one of `docsify`'s dependency: **marked**.

```
WS-2019-0024
More information
moderate severity
Vulnerable versions: >= 0.5.0, < 0.6.1
Patched version: 0.6.1

A flaw was found in nodejs-marked versions from 0.5.0 to before 0.6.1. 
Affected versions of this package are vulnerable to Regular Expression Denial of Service (ReDoS).
Input to the host variable is vulnerable when input contains parenthesis in link URIs, coupled with a high number of link tokens in a single line.
```

It was fixed in **docsify@4.10.0**: https://github.com/docsifyjs/docsify/pull/934.

# What this PR does

This PR bumps the version of docsify (previously >=3) to the version **^4.10.2** to fix the vulnerabilities audited in the project by npm.

The snapshots tests were updated accordingly.

### CODE REVIEW

File: e2e/index.js.md [hunk 1]
```diff
@@ -16,8 +40,8 @@ Generated by [AVA](https://ava.li).
       docsify start <path>  Server for SSR␊
     ␊
     Global Options␊
-      --help, -h     Show help                                             [boolean]␊
-      --version, -v  Show version number                                   [boolean]␊
+      --help, -h     Show help                                             [booléen]␊
```
Comment (anikethsaha) [inline e2e/index.js.md]: Can you please check why its coming 
`[booléen]␊`

the previous one was fine.

Comment (alexandre-lelain) [reply e2e/index.js.md]: Actually I'm investigating it because I have no clue why french is printed instead of english

Comment (anikethsaha) [reply e2e/index.js.md]: I guess no need to commit the snapshots

cc @jamesgeorge007 can you take a look why its happening

Comment (alexandre-lelain) [reply e2e/index.js.md]: Perhaps the types are translated according to the locale of the client - I'm in France which might explain why

Comment (alexandre-lelain) [reply e2e/index.js.md]: I'll remove this commit

Comment (anikethsaha) [reply e2e/index.js.md]: ok then it should not update the snapshots, it should throw error.
Something is wrong I guess with the test setup or with the npm script

Comment (jamesgeorge007) [reply e2e/index.js.md]: Yeah it isn't intended.

Comment (anikethsaha) [reply e2e/index.js.md]: I guess no need to commit the snapshots....

**Pull request merged**